server/main.py

```diff
@@ -31,14 +31,40 @@ class Payload(BaseModel):
     tvib: Optional[float] = None
     trot: Optional[float] = None
     path_length: float
-    simulate_slit: Optional[int] = None
+    simulate_slit: int
```

Comment:
I thought this should be an optional since we are not always applying a slit, so I don't think we'd always have a value passed here right?

Reply:
I still have a concern about the type change here